Android Sample App Consolidation and Kotlin Conversion

handstandsam opened the pull request:
It is confusing having two sample Android Applications.  This is a step in the right direction to move everything to Kotlin and cleanup the codebase.  There will still be more iterations that would make this ideal, but it's a good step forward.

Completed:
* Removed viewBinding
* Convert Original Sample App to Kotlin
* Use `local.properties` method of setting the `APP_KEY` for the original sample app
* Move over the code from the new sample app into the Android sample app.
* Migrated AsyncTasks to Kotlin Coroutines
* Fixed layout on smaller screens




<table>
<tr>
<td><img width="540" alt="Screen Shot 2022-08-12 at 2 45 20 PM" src="https://user-images.githubusercontent.com/264948/184423856-8e475236-ec3b-4499-9df3-cad48aff1701.png">
</td>
</tr>
</table>

rharter reviewed (approved):
This looks reasonable to me.  

Not a blocker, but I think the example could be easier for users to use if everything wasn't named `Dropbox*`, since that makes it really hard to peruse and understand what is part of the Dropbox API and what's part of the example (i.e. `DropboxApi` isn't actually part of the `Dropbox SDK` 😆 ).  Perhaps making the sample app `Sample` or something could help.

alisonthemonster reviewed (approved):
Looks good to me! Thank you!

handstandsam commented:
Good call @rharter .  I updated the naming to be clear.  Also I moved all the `*Task.kt` (async task leftovers) to the `DropboxApiWrapper.kt` file as a metho
<img width="369" alt="Screen Shot 2022-08-12 at 4 58 08 PM" src="https://user-images.githubusercontent.com/264948/184444331-66556986-1415-45e2-8868-e88865f0b6dc.png">
d. 😄
